doc/source/serve/doc_code/fault_tolerance/k8s_config.yaml

```
@@ -0,0 +1,187 @@
+# __start__
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: redis-config
+  labels:
+    app: redis
+data:
+  redis.conf: |-
+    dir /data
+    port 6379
+    bind 0.0.0.0
+    appendonly yes
+    protected-mode no
+    pidfile /data/redis-6379.pid
```

```suggestion
    port 6379
    bind 0.0.0.0
    protected-mode no
```
let's remove unnecessary details here

---

Good idea, I removed the unnecessary configs.

---

I did some extra testing. It looks like `requirepass 5241590000000000` is a required config here, so I'll include that.

---

Does this mean we can safely remove `protected-mode no` @simon-mo?

---

`protected-mode no` is needed so the redis container listen on 0.0.0.0 instead of localhost, which then allows service to hit it.